cnames_active.js

```diff
@@ -3010,6 +3010,7 @@ var cnames_active = {
   "sweetalert": "t4t5.github.io/sweetalert",
   "sweetconfirm": "koddr.github.io/sweetconfirm.js",
   "swipe": "lyfeyaj.github.io/swipe",
+  "swiss": "suisse.onrender.com"
```

Comment:
```suggestion
  "swiss": "suisse.onrender.com",
```
As the CI error notes, you're missing a comma at the end of your entry